src/main/scala/wordbots/ErrorAnalyzer.scala

```diff
@@ -9,6 +9,8 @@ import scala.util.{Failure, Success}
 case class ParserError(description: String, suggestions: Set[String] = Set())
 
 object ErrorAnalyzer {
+  import Semantics._
```

Comment:
What does this change do?

Reply:
I had to move all the semantic types into a `Semantics` object (think namespace, basically) for `Randomizer` to be able to do the compile-time magic correctly. So now `Semantics._` needs to be imported whenever I need the semantic types.